cfme/infrastructure/virtual_machines.py

```diff
@@ -634,11 +634,17 @@ def create(self, force_check_memory=False):
                 snapshot_dict['name'] = self.name
             if force_check_memory or self.vm.provider.mgmt.is_vm_running(self.vm.name):
                 snapshot_dict["snapshot_memory"] = self.memory
+            if force_check_memory and not view.snapshot_vm_memory.is_displayed:
+                raise NoSuchElementException('Snapshot VM memory checkbox not present')
```

Comment:
Without this, [test_memory_checkbox](https://github.com/ManageIQ/integration_tests/blob/88dd1c3e757215ac706975b15e4851ff28e01123/cfme/tests/infrastructure/test_snapshot.py#L94) is never passing.